solr/contrib/ltr/src/java/org/apache/solr/ltr/ranking/LTRCollector.java

```diff
@@ -125,7 +126,7 @@ public TopDocs topDocs(int start, int howMany) {
         System.arraycopy(topRerankDocs.scoreDocs, 0, scoreDocs, 0, howMany);
         topRerankDocs.scoreDocs = scoreDocs;
       }
-
+      log.info("mainCollector.topDocs: {} reRankRescorer.rescore: {}",(time2-time1), (time3-time2));
```

Comment:
do we want to put perf logging into master? If the answer is yes, should we have the ability to turn it on and off? (since perf logging itself can have a small impact on performance).